index.js

```diff
@@ -0,0 +1,29 @@
+const calculate = (arry) => {
+  const num1 = parseInt(calculateArry(arry)[0])
+  const num2 = parseInt(calculateArry(arry)[2])
+  const operator = (calculateArry(arry)[1])
```

Comment:
Doing it this way means you are looping through the array 3 times when you don't have to.

```JavaScript
const calculate = (rawOperation) => {
  const cleaned = cleanOperation(rawOperation)
  const num1 = Number(cleaned[0])
  const num2 = Number(cleaned[2])
  const operator = cleaned[1]
```